Makefile

```diff
@@ -28,6 +28,9 @@ pip: requirements.txt
 models:
 	mkdir -p $@
 	cd $@ && bash ./download-models.sh # $(TARBALLS)
+	cd deen/ende.student.tiny11
+	gunzip lex.s2t.gz # wasm build doesn't support zipped input
+	cd ../..
```

Comment:
Does this work, make launches commands in different shells right? cd && chaining or paranthesis is required, or am I missing something?

Reply:
I forgot that it's a makefile... fixd